inc0gnito-ctf-2015/.gitignore

```diff
@@ -0,0 +1,3 @@
+network/Hack-a-network-administrator's-terminal/network_hacking_102.zip
+forensic/POS/POS_9b648dbca2017349512c89466391c07d722418d1
+forensic/old_school/old_school_65c87efde678662c9139c81e016a62671cea2d6f
```

Comment:
hey can you do me a favor and add these files to google drive / sharing service 

it's taking a lot of time for me to do it (my bandwidth sucks )